scenarios/nomad-consul-vault-secure/README.md

```diff
@@ -0,0 +1,134 @@
+# Scenario: Nomad Consul Vault Quickstart
+This scenario deploys Nomad, Consul and Vault with security configurations in place. This scenario is useful when you have to play around the features of Nomad and Vault (with or without Consul ) with the security aspects and runs workload Identity for Vault.
+
+## Prerequisites
+The following tools are required to run these scenarios:
+
+- Shikari and Lima with Socket_VMNet configured
+- Requires a base VM image built using packer (../../packer/hashibox.pkr.hcl)
+- Uses qemu driver (you can use vz by modifying vmType in the template)
+- The vault config uses raft as storage.
+- Requires Packer build using Consul, Nomad and Vault binaries, may need to update the [variables.pkvars.hcl] file incase there's a need to change versions than default.
+- Update the vault.job.hcl to the correct shikari cluster name for datacenters block
+- This setup creates vault tokens for nomad jobs using workload Identity
+- It also supports Vault Workload Identity and is enabled by default when the Nomad version is `>=1.8.0`. If you need to disable Workload Identity configuration with versions `>=1.8.0`, please pass `-e NOMAD_CONSUL_WI=false` with `shikari create`.
```

Comment:
Why are these under the pre-requisites section?

Reply:
updated